tsconfig.json

```diff
@@ -1,28 +1,18 @@
 {
   "compilerOptions": {
-    "importHelpers": true,
     "allowJs": true,
-    "allowSyntheticDefaultImports": true,
     "declaration": false,
-    "jsx": "react",
-    "lib": [
-      "esnext",
-      "dom"
-    ],
-    "module": "esnext",
-    "moduleResolution": "node",
+    "module": "nodenext",
```

Comment:
I'm pretty sure this is the thing that actually fixed it. https://www.typescriptlang.org/tsconfig#module